crates/editor/src/editor.rs

```diff
@@ -2172,6 +2172,10 @@ impl Editor {
         show_completions: bool,
         cx: &mut ViewContext<Self>,
     ) {
+        cx.on_next_frame(|_, cx| {
```

Comment:
I placed one here because `selections_did_change` was updating the IME panel either with the previous bounds or the current bounds

Reply:
Ahh, so these are basically "check the bounds after we've rendered these changes"